format.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+
+def main():
+    files = [
+        os.path.join(dp, f)
+        for dp, dn, fn in os.walk(".")
+        for f in fn
+        if f.endswith(".py")
+    ]
+
+    for f in files:
+        subprocess.run(["python3", "-m", "black", "-q", f])
```

Comment:
Pedantic, but generally one would want to use `sys.executable` here instead (to accomodate virtualenvs and platform oddities).

However, given that black has a --check mode and traverses subdirectories, it seems better to just have users (and travis) execute black directly?